### PR TITLE
Change master to main

### LINKED ISSUE #12: Rename `master` branch to `main`
The new default branch name on GitHub is `main`. We should rename the `master` branch in this template repo to `main` for consistency. This will involve a few changes in the CI YAML files and utility scripts.

### PR DESCRIPTION
**Pull request by thomasyu888**
fixes #12. opening this PR to view the changes done.  After this is accepted, we can:

1.  Make `main` the default branch
2. delete `master`

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2020] [Sage Bionetworks]
```
Comment (BrunoGrandePhD) [inline LICENSE:189]: 😆 We're professional now!

**Review by BrunoGrandePhD — APPROVED** [review]
Thanks for taking this on, Tom!

**Comment by thomasyu888** [comment]
@BrunoGrandePhD  No problem, This PR actually depends on https://github.com/Sage-Bionetworks-Workflows/dockstore-tool-template/pull/24

**Review by BrunoGrandePhD — APPROVED** [review]
I confirmed that there are no remaining instances of `master` in this PR. 

🚀 

Whoever merges this PR should also change the default branch to `main` to avoid confusion and delete the `master` branch.

**Review by BrunoGrandePhD — APPROVED** [review]
Note to self: We should make sure that this version bumping in the template stops once we move to the new templating approach.

**Pull request merged**